winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,149 @@
+
+## Border
+
+Personalize the border of `AvatarView` control by adjusting the border brush and border thickness properties to achieve the desired look.
+
+### BorderBrush
+
+To set the color of the border surrounding your `AvatarView`, you can use the `BorderBrush` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView  ContentType="AvatarCharacter" 
+                          AvatarCharacter="Avatar16"
+                          AvatarSize="ExtraLarge"
+                          BorderBrush="Red">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarShape = AvatarShape.Circle;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar16;
+avatarView.BorderBrush = new SolidColorBrush(Colors.Red);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderBrush](avatarview_images/winui_borderbrush_avatarview.png)
+
+### Border Thickness
+
+You can customize the thickness of the AvatarView control by using the `BorderThickness` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView   AvatarShape="Custom"
+                           Width="100"
+                           Height="100"
+                           CornerRadius="50"
+                           BorderBrush="Black" 
+                           BorderThickness="4">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarShape = AvatarShape.Custom;
+avatarView.Height = 100;
+avatarView.Width = 100;
+avatarView.BorderThickness = new Thickness(4);
+avatarView.CornerRadius = new CornerRadius(50);
+avatarView.BorderBrush = new SolidColorBrush(Colors.Black);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_borderthickness_avatarview.png)
+
+### Background
+
+You can customize the background of the AvatarView control by using the `Background` property.
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom background](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+### Gradients
+
```

Comment:
@MadhumithanJayapathi ## Gradient background

Reply:
@Manivannan-E Changed the ### Gradient to ##Gradient